engine/src/main/java/org/terasology/config/AudioConfig.java

```diff
@@ -31,39 +31,63 @@
 
     private transient PropertyChangeSupport propertyChangeSupport = new PropertyChangeSupport(this);
 
+    /**
+     * @return Returns a float of the sound volume
+     */
     public float getSoundVolume() {
         return soundVolume;
     }
 
+    /**
+     * @param soundVolume Sets the sound volume starting from the previous value and going to the soundVolume passed in
```

Comment:
The way it's written here I'm interpreting it that this _ramps up_ the sound volume linearly from the current value to the set value _over time_, but the code below doesn't support this interpretation. I'd simply write:

"a float representing a new sound volume, with 0 meaning no sound and 100 (or 1?) meaning the highest volume possible."